.github/workflows/zap_test.yml

```diff
@@ -65,6 +69,7 @@ jobs:
       - name: Build and run dev container task
         uses: devcontainers/ci@v0.3
         with:
+          subFolder: products/db-zap-opendata
```

Comment:
Is this test still needed if zap doesn't have its own dev container at this point?

Reply:
needed, no. cool, yes.

[the devcontainer files](https://github.com/NYCPlanning/data-engineering/tree/main/products/db-zap-opendata/.devcontainer) are still here and since I only disabled the BigQuery stages of the ZAP export and MapZAP hasn't been converted from BigQuery to Postgres yet, it didn't seem worth deleting everything yet

Reply:
I hope we can convince @AmandaDoyle that coolness of codebase should be one of our KPIs. Maybe the only one!